.circleci/config.yml

```diff
@@ -76,6 +76,7 @@ jobs:
             yarn version --patch
             yarn publish --access public
             git add package.json
+            git add yarn.lock
```

Comment:
@Itsindigo @vladovidiu If your yarn lock is after it hits your CI server you're doing something wrong :)